cartridges/plugin_beam/cartridge/templates/default/beam/beam_select_nonprofit.isml

```diff
@@ -33,6 +37,60 @@
 
     initNetworkListeners(); // Will emit Beam network call events whenever the site makes an AJAX call
 
+    async function updateCustomAttributes(payload) {
+        try {
+
+            const url = `/on/demandware.store/Sites-${currentSiteID}-Site/${currLocale}/Cart-UpdateBeamCustomAttributes`
```

Comment:
`suggestion` I think this line was updated, though want to call out that there may be situations that this isn't going to be the right URL. It _might_ be okay for now, but I think generating the URL where we're doing all the other `actionUrls` and getting the URL it via `pdict` could be more resilient.

https://salesforcecommercecloud.github.io/b2c-dev-doc/docs/current/scriptapi/html/index.html?target=class_dw_web_URLAction.html

https://github.com/Beam-Impact/beam-salesforce-cartridge/blob/96ec28e96231966cf613889b37eed829ec481645/cartridges/plugin_beam/cartridge/models/cart.js#L86